play-services-core/src/main/res/values/strings.xml

```diff
@@ -43,6 +43,11 @@ This can take a couple of minutes."</string>
     <string name="auth_action_notification_title">Account action required</string>
     <string name="auth_action_notification_content">Your Google account needs additional setup.</string>
 
+    <string name="auth_action_reauth_notification_title">Account Issue Detected</string>
+    <string name="auth_action_reauth_notification_content">There is an issue with your account (%1$s). Please log in again.</string>
+    <string name="auth_action_reauth_notification_re_login">Log In Again</string>
+    <string name="auth_action_reauth_notification_remove_account">Remove Account</string>
```

Comment:
```suggestion
    <string name="auth_action_reauth_notification_remove_account">Remove account</string>
```

Basically try not to use title case, it can also end up being a bit annoying when translating because it's not always obvious what's actually meant to be like that and what isn't: https://m3.material.io/foundations/content-design/style-guide/ux-writing-best-practices (check "Use sentence case"